tests/to-json-tests.js

```diff
@@ -849,6 +849,80 @@ test('surfaces fallback text for table with any cell with more than max characte
             }
         ],
         "actions": [],
-        "version": "1.0"
+        "version": "1.2"
+    });
+});
+
+
+test('can handle code text', t => {
+    var result = AdaptiveHtml.toJSON(`
+        <code>normal code text</code>
+    `);
+    t.deepEqual(result, {
+        "type": "AdaptiveCard",
+        "body": [
+            {
+                type: "TextBlock",
+                text: "normal code text",
+                wrap: true
+            }
+        ],
+        "actions": [],
+        "version": "1.2"
+    });
+});
+
+test('can handle guru code snippet', t => {
+    var result = AdaptiveHtml.toJSON(`
+        <code data-ghq-card-content-type="CODE_SNIPPET">code snippet text</code>
+    `);
+    t.deepEqual(result, {
+        "type": "AdaptiveCard",
+        "body": [
+            {
+                type: "RichTextBlock",
+                inlines: [
+                    {
+                        type: "TextRun",
+                        fontType: "monospace",
+                        highlight: true,
+                        text: "code snippet text",
+                        wrap: true
+                    }
+                ],
+            }
+        ],
+        "actions": [],
+        "version": "1.2"
```

Comment:
theres inconsistency here with the use of quotes on field names.

Reply:
true, i've seen inconsistency in the file overall (could be useful cleanup in a later PR) 😢 do you have a preference for one way over the other?

Reply:
cool! i can update the tests i wrote for this PR and put in a backlog item for the remaining cleanup

Reply:
sounds good!